includes/Core/Authentication/Google_Proxy.php

```diff
@@ -74,6 +74,55 @@ public function url( $path = '' ) {
 		return $url;
 	}
 
+	/**
+	 * Gets site fields.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @return array
+	 */
+	public function get_site_fields() {
+		return array(
+			'name'       => wp_specialchars_decode( get_bloginfo( 'name' ) ),
+			'url'        => home_url(),
+			'admin_root' => admin_url(),
```

Comment:
This shouldn't be `admin_url()`, but continue to be what it was before.

Reply:
What is `admin_root` used for? It seems unnecessary if we're passing full admin URLs for return and action URIs.

Reply:
It is unnecessary, but it's not certain we can get the enhancement for the new parameters live on the proxy before this release gets out, so we need to leave `admin_root` in there for now. It can be removed in the following release probably :)

Reply:
Oh okay, that makes sense 👍

Reply:
Done!